.goreleaser.yaml

```diff
@@ -0,0 +1,53 @@
+# Make sure to check the documentation at https://goreleaser.com
+
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 1
+
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+
+builds:
+  - main: ./cmd/world
+    binary: world
+    env:
+      - CGO_ENABLED=0
+      - SENTRY_DSN={{ if index .Env "SENTRY_DSN"  }}{{ .Env.SENTRY_DSN }}{{ else }}default_null{{ end }}
+      - POSTHOG_API_KEY={{ if index .Env "POSTHOG_API_KEY"  }}{{ .Env.POSTHOG_API_KEY }}{{ else }}default_null{{ end }}
+    ldflags:
+      - -s -w
+      - -X main.AppVersion={{.Version}}
+      - -X main.SentryDsn={{.Env.SENTRY_DSN}}
+      - -X main.PosthogAPIKey={{.Env.POSTHOG_API_KEY}}
+    goos:
```

Comment:
Add apple silicon build

Reply:
:heavy_check_mark: the default goreleaser config already create builds for [386, amd64, arm64], which includes darwin-arm64 for Apple Silicon. I'm adding the GOARCH variable here to show that explicitly. :+1: